de.tubs.variantsync.core/src/de/tubs/variantsync/core/managers/data/ConfigurationProject.java

```diff
@@ -75,13 +76,14 @@ public void addVariant(IProject project) {
 
 	public Configuration getConfigurationForVariant(IProject project) {
 		if (project != null) {
-			for (IFile configPath : configurationProject.getAllConfigurations()) {
+			for (Path confPath : configurationProject.getAllConfigurations()) {
+				IFile configPath = (IFile) EclipseFileSystem.getResource(confPath);
 				String configFileName = configPath.getName();
 				String configName = configFileName.substring(0, configFileName.lastIndexOf('.'));
 				System.out.println("[ConfigurationProject.getConfigurationForVariant] Check name equality Project(" + project.getName() + ") with Config(" + configName + ")");
 				if (configName.equals(project.getName())) {
 					ConfigurationManager configurationManager = ConfigurationManager.getInstance(Paths.get(configPath.getRawLocationURI()));
-					if (configurationManager != null) return configurationManager.getObject();
+					if (configurationManager != null) return configurationManager.getObject();			
```

Comment:
This is against basic style guides, all optional brackets should be written. @pmbittner I thought you copied the FeatureIDE style config to VariantSync?

Reply:
I did but I don't know how if these files are considered by IntelliJ and I don't know if you have to import them manually in Eclipse and I don't think the style files will make the IDE rewriting existing code on its own.

Reply:
We had to import the code style xml file (Settings -> Java -> CodeStyle -> Formatter -> Import) first. 
We will fix this in future and rerun the code formatter on all other files.

Reply:
In Intellij File -> Settings -> Editor -> Code Style -> Java -> import eclipse scheme